dallinger/experiment.py

```diff
@@ -492,6 +494,25 @@ def run(self, exp_config=None, app_id=None, bot=False, **kwargs):
             )
         return self._finish_experiment()
 
+    def retrieve(self, app_id, exp_config=None, bot=False, **kwargs):
```

Comment:
How about naming this "collect"? The goal is to find a verb that works for either action: retrieving an existing data set or running the experiment to generate a new data set.